src/Microsoft.Azure.AppConfiguration.AspNetCore/AzureAppConfigurationRefreshExtensions.cs

```diff
@@ -23,14 +24,21 @@ public static IApplicationBuilder UseAzureAppConfiguration(this IApplicationBuil
                 throw new ArgumentNullException(nameof(builder));
             }
 
+            IConfigurationRefresherProvider refresherProvider = (IConfigurationRefresherProvider)builder.ApplicationServices.GetService(typeof(IConfigurationRefresherProvider));
+
             // Verify if AddAzureAppConfiguration was done before calling UseAzureAppConfiguration.
             // We use the IConfigurationRefresherProvider to make sure if the required services were added.
-            if (builder.ApplicationServices.GetService(typeof(IConfigurationRefresherProvider)) == null)
+            if (refresherProvider == null)
             {
                 throw new InvalidOperationException("Unable to find the required services. Please add all the required services by calling 'IServiceCollection.AddAzureAppConfiguration' inside the call to 'ConfigureServices(...)' in the application startup code.");
```

Comment:
There is no `ConfigureServices` anymore since .NET 6. If you prefer, we can change it in a separate PR.

Reply:
I guess it is a pretty small change, would it just be removing the part saying `inside the call to 'ConfigureServices(...)'`?